packages/core/useEventSource/index.ts

```diff
@@ -0,0 +1,35 @@
+import { ref, onMounted, onUnmounted, Ref } from 'vue-demi'
+
+export function useEventSource(url: string) {
+  const data: Ref<any> = ref(null)
+  const state = ref('CONNECTING') as Ref<'OPEN' | 'CONNECTING' | 'CLOSED'>
+  let es: EventSource
+  const close = () => {
+    es?.close()
+  }
+
+  onMounted(() => {
+    es = new EventSource(url)
+    es.onopen = () => {
+      state.value = 'OPEN'
+    }
+
+    es.close = es.onerror = () => {
```

Comment:
> EventSource does an auto-reconnect if the connection is not explicitly closed.

So I guess you should call `close` in this function to prevent it to auto reconnect?